diesel_cli/tests/print_schema.rs

```diff
@@ -41,6 +41,11 @@ fn print_schema_with_foreign_keys() {
     test_print_schema("print_schema_with_foreign_keys", vec![]);
 }
 
+#[test]
+fn print_schema_column_renaming() {
+    test_print_schema("print_schema_column_renaming", vec![]);
```

Comment:
Will need "--with-docs" as argument on master!

(And this is something bors would catch too)